### ISSUE TITLE
ss04 variant shows some wrong glyphs after an italic block in Visual Studio (WPF?)

### ISSUE THREAD

**Issue by marklam** [be5invis/Iosevka#1341]
In Visual Studio (so possibly any WPF rendering), some of the glyphs are rendered incorrectly for a non-italic block following an italic block.

As an example, the [Markdown editor extension](https://github.com/MadsKristensen/MarkdownEditor2022) (although other extensions which render italics in the editor also do this) will render 
```markdown
*a* abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ0123456789
```
as something like
```text
*𝘢* ab▒d▒fghijklmnopqrstuvwxyzAB🤐DEFGHIJKLMNOPQRSTUVWXYZ0123456789
```
i.e. lower-case c and e, and upper-case C are wrong after the 'a' which is shown in italics

![image](https://user-images.githubusercontent.com/1962640/165501818-98f11782-3749-4869-b07e-894a13af884a.png)

I've checked that it doesn't happen with the Consolas font, and that the character codes in the source file aren't actually corrupted (copy-pasting to notepad gives the correct text).

---

* Your font version: 15.2.0
* Your font variant: ss04
* Your operating system (name and version): Windows 10
* Your application using Iosevka: Visual Studio

**Comment by be5invis** [comment]
Maybe related with caching. You could try 
* Uninstalling all the fonts
* Restart PC
* Then reinstall them, and see whether it get fixed.

**Comment by marklam** [comment]
I gave this a go - I uninstalled iosevka-slab and iosevka-ss04, rebooted and then reinstalled iosevka-ss04. 
Sadly it still looks the same.

**Comment by marklam** [comment]
I 've checked it on a Windows 10 machine that's never had the font installed before, using a [repro app that shows the problem](https://github.com/marklam/IosevkaWpfTextRepro/releases/tag/demo). The source is there too if you want to be sure of what you're running (Windows Defender understandably wants to discourage you from running random unsigned executables from the internet).

The font file I used was [super-ttc-iosevka-ss04](https://github.com/be5invis/Iosevka/releases/download/v15.2.0/super-ttc-iosevka-ss04-15.2.0.zip)

**Comment by be5invis** [comment]
Maybe you should try the TTF packages instead.

**Comment by marklam** [comment]
Yes, extracting & installing those worked. (It's only a bit less convenient, since you can multi-select all the extracted files and right-click install.)

Thanks for the help.

**Comment by be5invis** [comment]
Resolve as external. Maybe related with TTC support...
